custom_model_runner/datarobot_drum/resource/components/Python/uwsgi_component/uwsgi_serving.py

```diff
@@ -145,6 +145,10 @@ def predict(self, url_params, form_params):
                 # this counter is managed by uwsgi
                 self._total_predict_requests.increase()
                 self._predict_calls_count += 1
+        except Exception as ex:
+            self._logger.error(ex)
+            response_status = HTTP_513_DRUM_PIPELINE_ERROR
```

Comment:
@yakov-g raised a good point - why 513 and not 500?
513 is designed only for errors happened during drum initilization.
prediction errors should be 500

Reply:
also, looks like this ticket is about this issue
could you please have a look at close it if it's fixed https://datarobot.atlassian.net/browse/RAPTOR-3751?

Reply:
thanks
I'll take a look, if @zohar-mizrahi won't